### ISSUE TITLE
Cannot set image alternative for accessibility

### ISSUE THREAD

**Issue by correi-f** [theopenconversationkit/tock-react-kit#98]
In cards, the attribute alt of the image is always set with the title of the card. Sometimes images show more than a title and we cannot describe it without changing the title.

Example :
![image](https://user-images.githubusercontent.com/2005914/172787115-aaef46da-8419-4336-8880-79348657f05f.png)
![image](https://user-images.githubusercontent.com/2005914/172787178-652719e0-998a-47e2-bdaf-ecc58bb3c3fb.png)

This issue is linked to another issue in Tock and Tock Studio in order to give an extra parameter description of a file.
